src/main/webapp/WEB-INF/view/activityfeed.jsp

```diff
@@ -32,14 +32,19 @@
     <% } %>
     <a href="/about.jsp">About</a>
     <a href="/activityfeed">Activity Feed</a>
+    <%  if (request.getSession().getAttribute("user") != null) { %>
```

Comment:
Could we potentially combine this if statement with the if statement determining whether to show the login? It looks like they are checking the same conditions, so we might be able to just check those conditions once instead, unless you intentionally wanted the profile to show up in this order in the nav bar, which is also okay since sometimes Profile is in the top right corner in a lot of applications.

Reply:
You're right, I unintentionally did that. How redundant! I just decided to combine it with the initial if statement.